src/main/resources/io/jenkins/plugins/extratoolinstallers/installers/Messages.properties

```diff
@@ -38,3 +38,18 @@ IsAlreadyOnPath.versionPatternDidNotMatch=No version number found within command
 IsAlreadyOnPath.versionIsOk=Version "{0}" is acceptable.
 IsAlreadyOnPath.versionIsTooLow=Version "{0}" is too low; must be "{1}" or greater.
 IsAlreadyOnPath.versionIsTooHigh=Version "{0}" is too high; must be "{1}" or under.
+
+IsAlreadyInGivenDirectory.DescriptorImpl.displayName=Check tool is already installed to the given directory
+IsAlreadyInGivenDirectory.agentIsOffline=Agent is not online
+IsAlreadyInGivenDirectory.executablePathIsEmpty=Executable Path field is empty
+IsAlreadyInGivenDirectory.noVersionValidation=No version validation will be performed.
+IsAlreadyInGivenDirectory.versionCmdIsEmpty=Version command field is empty.
+IsAlreadyInGivenDirectory.versionCmdContainsSpaceButHasNoArguments=Version command must separate the executable file path from its arguments using a newline, not a space.
+IsAlreadyInGivenDirectory.versionPatternIsEmpty=Version pattern field is empty.
+IsAlreadyInGivenDirectory.versionPatternIsInvalid=Version pattern "{0}" is not valid.
+IsAlreadyInGivenDirectory.versionMinMaxNotSpecified=Neither minimum or maximum version has been provided.
+IsAlreadyInGivenDirectory.versionMaxMustNotBeLessThanMinimum=Maximum version must not be less than minimum "{0}".
+IsAlreadyInGivenDirectory.versionPatternDidNotMatch=No version number found within command output.
+IsAlreadyInGivenDirectory.versionIsOk=Version "{0}" is acceptable.
+IsAlreadyInGivenDirectory.versionIsTooLow=Version "{0}" is too low; must be "{1}" or greater.
+IsAlreadyInGivenDirectory.versionIsTooHigh=Version "{0}" is too high; must be "{1}" or under.
```

Comment:
Please add a linefeed at the end of the file.
(it won't change the code behaviour, but it'll make github happier, make subsequent changes that add to the file clearer, and that'll make me happier).